package.json

```diff
@@ -3,6 +3,7 @@
   "scripts": {
     "dev": "vite --https",
     "build": "vite build",
+    "start": "vite --https & php artisan serve && fg",
```

Comment:
I personally don't need this step. `yarn dev` is enough on my machine. What's `fg`?

Reply:
If I am not running artisan serve nothing works for me.

When I run vite it opens the vite page, but doesn't actually provide me with any frontend. The frontend for me shows up on the localhost:8000 page that php artisan serve spits out.

Reply:
Oh god, I truly am not the guy to mess around with php. Yep. Works fine on my side with just yarn dev as well. But it starts on port 8000, which is the same as the default of php artisan serve. Hence, I though the php artisan serve was making it work, while really the APP_URL env var should be updated.